sympy/core/relational.py

```diff
@@ -124,6 +124,28 @@ def reversed(self):
         a, b = self.args
         return ops.get(self.func, self.func)(b, a, evaluate=False)
 
+    @property
+    def negated(self):
+        """Return the negated relationship.
+
+        Examples
+        ========
+
+        >>> from sympy import Eq
+        >>> from sympy.abc import x
+        >>> Eq(x, 1)
+        Eq(x, 1)
+        >>> _.negated
+        Ne(x, 1)
+        >>> x < 1
+        x < 1
+        >>> _.negated
+        x >= 1
+        """
+        ops = {Eq: Ne, Ge: Lt, Gt: Le, Le: Gt, Lt: Ge, Ne: Eq}
+        a, b = self.args
+        return ops.get(self.func, self.func)(a, b, evaluate=False)
```

Comment:
The second self.func is a bad idea.

Two alternatives:
1. Live with a NoneType-error if a new Relational is introduced but not handled here.
2. Use a lambda with ~. I have the code, but somehow seems not required as I cannot really see that there are more Relational out there...

Reply:
> 1. Live with a NoneType-error if a new Relational is introduced but not handled here.

@asmeurer knows the details better, but maybe this is preferable:

```python
for k in ops:
    if isinstance(self.func, k):
        return ops[k](a, b, evaluate=False)
raise ValueError('unrecognized %s' % self.func)
```